name.abuchen.portfolio.tests/src/name/abuchen/portfolio/util/TradeCalendarTest.java

```diff
@@ -159,66 +158,97 @@ public void testMovingHolidays()
         assertThat(calendar.isHoliday(LocalDate.parse("2021-04-02")), is(true));
+
+        // Juneteenth
+        assertThat(calendar.isHoliday(LocalDate.parse("2020-05-25")), is(true));
         assertThat(calendar.isHoliday(LocalDate.parse("2021-05-31")), is(true));
+
+        // Independence Day
+        assertThat(calendar.isHoliday(LocalDate.parse("2020-07-03")), is(true));
         assertThat(calendar.isHoliday(LocalDate.parse("2021-07-05")), is(true));
+        
+        // Labour day
         assertThat(calendar.isHoliday(LocalDate.parse("2021-09-06")), is(true));
+        assertThat(calendar.isHoliday(LocalDate.parse("2022-09-05")), is(true));
+        assertThat(calendar.isHoliday(LocalDate.parse("2023-09-04")), is(true));
+
+        // Thanksgiving
         assertThat(calendar.isHoliday(LocalDate.parse("2021-11-25")), is(true));
+        assertThat(calendar.isHoliday(LocalDate.parse("2022-11-24")), is(true));
+        assertThat(calendar.isHoliday(LocalDate.parse("2023-11-23")), is(true));
+
+        // Chrismas
+        assertThat(calendar.isHoliday(LocalDate.parse("2020-12-25")), is(true));
         assertThat(calendar.isHoliday(LocalDate.parse("2021-12-24")), is(true));
+
+        // Special Holidays
+        assertThat(calendar.isHoliday(LocalDate.parse("1994-04-27")), is(true)); // President Nixon
+        assertThat(calendar.isHoliday(LocalDate.parse("2004-06-11")), is(true)); // President Reagan
+        assertThat(calendar.isHoliday(LocalDate.parse("2007-01-02")), is(true)); // President Ford
+        assertThat(calendar.isHoliday(LocalDate.parse("2012-10-29")), is(true)); // Hurricane Sandy
+        assertThat(calendar.isHoliday(LocalDate.parse("2012-10-30")), is(true)); // Hurricane Sandy
+        assertThat(calendar.isHoliday(LocalDate.parse("2018-12-05")), is(true)); // President Bush Sr.
+
+        // terrorist attacks (9/11)
+        assertThat(calendar.isHoliday(LocalDate.parse("2001-09-10")), is(false)); // open market
+        assertThat(calendar.isHoliday(LocalDate.parse("2001-09-11")), is(true)); // closed after terrorist attacks (9/11)
+        assertThat(calendar.isHoliday(LocalDate.parse("2001-09-12")), is(true)); // closed after terrorist attacks (9/11)
+        assertThat(calendar.isHoliday(LocalDate.parse("2001-09-13")), is(true)); // closed after terrorist attacks (9/11)
+        assertThat(calendar.isHoliday(LocalDate.parse("2001-09-14")), is(true)); // closed after terrorist attacks (9/11)
+        assertThat(calendar.isHoliday(LocalDate.parse("2001-09-17")), is(false)); // reopen market (Monday)
     }
 
     @Test
-    public void testLSE()
+    public void testTradeCalenderLSE()
     {
         // See https://www.londonstockexchange.com/trade/trading-access/business-days
 
         TradeCalendar calendar = TradeCalendarManager.getInstance("lse");
 
-        assertThat(calendar.isHoliday(LocalDate.parse("2019-01-01")), is(true));
```

Comment:
Removed despite claiming that no tests were removed.

Reply:
https://github.com/buchen/portfolio/blob/718f13d64d6f4d6526de484c948a4c4064f393b1/name.abuchen.portfolio.tests/src/name/abuchen/portfolio/util/TradeCalendarTest.java#L312
Look here and is not removed